Detectors/MUON/MID/Clustering/src/Clusterizer.cxx

```diff
@@ -200,22 +183,27 @@ bool Clusterizer::init(std::function<void(size_t, size_t)> func)
 }
 
 //______________________________________________________________________________
-void Clusterizer::makeCluster(const MpArea& areaBP, const MpArea& areaNBP, const int& icolumn, const int& deIndex)
+void Clusterizer::makeCluster(const MpArea& areaBP, const MpArea& areaNBP, uint8_t deId)
+{
+  double sqrt12 = 3.4641016;
```

Comment:
I assume this is an optimisation backed-up by some measurement ? ;-) 
Anyway, while at it, I would constexpr it